analyze_openapi_changes.py

```
@@ -0,0 +1,114 @@
+import os
+import json
+import argparse
+from datetime import datetime
+from deepdiff import DeepDiff
+from typing import Dict, Any, Optional
+
+def load_json_file(file_path: str) -> Optional[Dict[str, Any]]:
+    try:
+        with open(file_path, 'r') as f:
+            return json.load(f)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)